ClassBuilder/patternbuilder/Laptop.java

```diff
@@ -54,6 +54,6 @@ public String toString() {
     }
 }
 
-enum Processor {Intel, AMD};
+enum Processor {Intel_Const, AMD_Const};
 
-enum OS {Windows, Linux};
\ No newline at end of file
+enum OS {Windows_Const, Linux_Const};
```

Comment:
public enum Processor { INTEL, AMD }
public enum Os {WINDOWS, LINUX }
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

after last braces should be an empty line